lib/node/fullnode.js

```diff
@@ -434,8 +434,14 @@ class FullNode extends Node {
 
   async getMetaByAddress(addrs) {
     const mempool = this.mempool.getMetaByAddress(addrs);
+
+    if (this.plugins.bindex) {
+      const index = await this.plugins.bindex.getMetaByAddress(addrs);
+      return mempool.concat(index);
+    }
+
     const chain = await this.chain.getMetaByAddress(addrs);
-    return chain.concat(mempool);
+    return mempool.concat(chain);
```

Comment:
Why reverse this?

Reply:
This case would occur if indexes are disabled, so I think it should just return mempool actually. Will check that.